Runtime/MeshSimplifier.cs

```diff
@@ -67,17 +67,9 @@ public sealed class MeshSimplifier
         #endregion
 
         #region Fields
-        private bool preserveBorderEdges = false;
-        private bool preserveUVSeamEdges = false;
-        private bool preserveUVFoldoverEdges = false;
-        private bool preserveSurfaceCurvature = false;
-        private bool enableSmartLink = true;
-        private int maxIterationCount = 100;
-        private double agressiveness = 7.0;
+        SimplificationOptions simplificationOptions = SimplificationOptions.Default;
```

Comment:
Normally I would have requested that the private modifier should be added here, to keep the consistency with the rest of the code. I know that it's optional, but personally I always prefer to be more explicit.
But I'll approve this anyways.

Reply:
Missed that one. We leave out C# defaults w/ our code conventions here at Unity, but I usually try to match where I am contributing to.